frontend/javascripts/oxalis/model/reducers/volumetracing_reducer_helpers.ts

```diff
@@ -47,22 +47,19 @@ export function setActiveCellReducer(state: OxalisState, volumeTracing: VolumeTr
     activeCellId: id,
   });
 }
-export function createCellReducer(state: OxalisState, volumeTracing: VolumeTracing, id?: number) {
-  if (id === 0) {
-    // cellId 0 means there is no annotation, so there must not be a cell with id 0
-    return state;
-  }
```

Comment:
Was this check no longer needed?

Reply:
The previous signature allowed to pass an explicit `id` which should be set for the active segment id. This was only used by the `INITIALIZE_VOLUMETRACING` reducer. I think, this made the code a bit hard to understand and also goes against the name of `createCellReducer`.

Therefore, I adapted the function so that its only purpose is to really create a new cell id. Therefore, the old `id` parameter doesn't exist, anymore, and doesn't need to be checked against `0` anymore. Hope that makes sense :)